tests/thickness/InterpolatedStructure/test_interpolated_structure.py

```diff
@@ -272,6 +273,8 @@ def test_compute(
     result = interpolated_structure_thickness.compute(
         units, stratigraphic_order, basal_contacts, samples, map_data
     )
+    print('HERE', result.dtypes)
```

Comment:
Is this necessary for the test?

Reply:
Definitely typo. Removed in https://github.com/Loop3D/map2loop/pull/109/commits/ae22adc28cd7cce465be3158b79bee23414f016b